shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -292,10 +294,14 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
    *
    * This callback is not present in API < 20, which means lower API devices will see
    * the wider than expected padding when the status and navigation bars are hidden.
+   * The annotations to suppress "InlinedApi" and "NewApi" lints prevent lint warnings
```

Comment:
really minor nit: This is more of an implementation detail and not really documentation to callers of this method about what it should do. It makes more sense to move it to around the `@SuppressLint` annotation and comment with `// ` instead of Javadoc (`/** */`).